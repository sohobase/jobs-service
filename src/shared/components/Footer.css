@import "../styles/colors.css";
@import "../styles/fonts.css";
@import "../styles/variables.css";

:root {
  --footer-offset: calc(var(--offset) * 2) var(--offset);
}

.footer {
  padding: var(--footer-offset);
  font-size: var(--font-size-tiny);
  color: var(--color-text-secondary);
  text-align: center;
}
