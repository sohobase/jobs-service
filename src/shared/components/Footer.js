import React from 'react';
import { Link } from 'react-router';
import style from './Footer.css';

export default () => (
  <footer className={style.footer}>
    copyright
  </footer>
);
