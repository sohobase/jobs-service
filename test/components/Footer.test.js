import React from 'react';
import renderer from 'react-test-renderer';

import Footer from '../../src/shared/components/Footer';

describe('<Footer />', () => {
  // -- Renders
  it('renders by default', () => {
    const tree = renderer.create(<Footer />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
