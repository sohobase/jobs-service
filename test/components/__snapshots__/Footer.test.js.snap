exports[`<Footer /> renders by default 1`] = `
<footer
  className="footer">
  <a
    onClick={[Function]}
    style={Object {}}>
    Site Terms
  </a>
   & 
  <a
    onClick={[Function]}
    style={Object {}}>
    Conditions
  </a>
   for use and our Privacy Policy apply. ©2017 Sohobase Ltd. All References marked with a ™ or ® are trademarks of Sohobase. All rights reserved.
</footer>
`;
